### ISSUE TITLE
Can mammoth read word documents created with html-docx-js?

### ISSUE THREAD

**Issue by carlelieser** [mwilliamson/mammoth.js#71]
html-docx-js gets the given html and renders it as a word document. When reading this created document with mammoth, i get this: `An unrecognised element was ignored: w:altChunk`

Any ideas on how to get passed this? Can mammoth be used to do what it's meant for but reverse? Can mammoth create word documents? Anyways that'd be pretty cool. I figured out the reason the code earlier wasn't working was because I kept getting that error. It didn't show up on the console, but it did show up on the mammoth example where the error messages go. Mammoth works perfectly fine with chrome apps, thanks for this and I hope  you can improve it better by allowing it to read documents created with [html-docx-js](https://github.com/evidenceprime/html-docx-js)!

**Comment by mwilliamson** [comment]
html-docx-js seems to include HTML using `w:altChunk` elements. I'd need to see how the HTML is actually embedded in the Word document to see how straightforward adding support would be, and I'm not sure when I'll get time to take a look. If you could provide an example file, that would be helpful.

**Comment by carlelieser** [comment]
[Forest.docx](https://github.com/mwilliamson/mammoth.js/files/177106/Forest.docx)

Ok this is one of those files that gets the error

**Comment by MCTaylor17** [comment]
@carlelieser Your content for Forest.docx: 
```html
<!DOCTYPE HTML><head><title></title></head><body><div><br></div></body></html>
```
It appears as though altChunk files actually contain the HTML that they represent.  You can see this by opening it with a text editor and searching for `quoted-printable
Content-Location: file://\\fake\document.html` assuming it was produced with html-docx-js.

It does appear that the html is ever-so-slightly modified though.  For example, I'm looking at one of my documents and it seems as though all `=` signs are now `=3D`.  This effectively breaks attributes and hyperlinks, so the changes are critical and would need to be managed.  That said, it's should be possible to snip out the HTML and work with it directly.

Alternatively, you can **open the document in Word and "Save As"** to convert it to something mammoth is capable of reading.

**Comment by carlelieser** [comment]
Working on an update for [Writer](https://chrome.google.com/webstore/detail/writer/hlddiopdeghmcmdjjmpdegemnojihpib?hl=en-US), I managed to remedy the issue by checking if the provided .docx file had `<w:altChunk r:id="htmlChunk" />` in it. If so, the substring method was used to retrieve the text in between html body tags, effectively returning the HTML.

Not sure exactly how you would be able to implement this into mammoth, but that's basically how I did it. Also, I'd like to note that the whole system would pretty much go to shit if someone were to explicitly say '<w:altChunk r:id="htmlChunk" />' in an actual Word document created by Word, so I don't know how you'd handle that. Just wanted to update the issue, hope this helps.
